### PR TITLE
semih week-4

### PR DESCRIPTION
**Pull request by sigma21**

### CODE REVIEW

File: homework-submission/client/src/Week2.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                "done" : false
+            },
+        ]
+        
+    };
+};
+
+createToDo = () => {
+    const todos = this.state.todos;
+    todos.push({"description":"seriously, add some css! :)"});
```
Comment (Meno-101) [inline homework-submission/client/src/Week2.js]: Please have a look to the previous homework reviews.

File: homework-submission/client/src/Week2.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    this.setState({todos:todos});    
+};
+
+removeToDo = () => {
+    const todos = this.state.todos;
+    todos.pop();
```
Comment (Meno-101) [inline homework-submission/client/src/Week2.js]: Same here

File: homework-submission/client/src/Week2.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    this.setState({todos:todos});    
+};
+
+submitFormHandler = event => {
+    event.preventDefault();
+    const newItem = this.refs.todo.value
+    const todos = this.state.todos;
+    todos.push({"description":newItem,"done":false});
```
Comment (Meno-101) [inline homework-submission/client/src/Week2.js]: Same here!

File: homework-submission/client/src/Week2.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    this.setState({todos:todos});   
+};
+
+markAsDone = event => {
+    event.preventDefault();
+    const markasdoneIndex = this.refs.markasdone.value-1
+    const todos = this.state.todos;
+    todos[markasdoneIndex].done = true;
```
Comment (Meno-101) [inline homework-submission/client/src/Week2.js]: Same here

File: homework-submission/client/src/Week2.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+  render() {
+    const todos = this.state.todos;
+    return (
+      <div className="week2">
+        <ol>
+        {todos.map((item,index)=>
+            {return <li>
+            <pre id={index} ref={index} style={{textDecoration: item.done ? 'line-through' : 'none'}} 
+            onClick={()=>{}}>{item.description}</pre></li>})}
```
Comment (Meno-101) [inline homework-submission/client/src/Week2.js]: Here too.

**Comment by wouterkleijn** [comment]
Hey @sigma21 Can you please make the necessary changes? Thanks!

**Pull request closed without merging**